.markdownlint.json

```diff
@@ -0,0 +1,9 @@
+# THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
```

Comment:
Not related to this PR, but I wonder why `markdown` lint is fine having these `#` comments here :)
This doesn't look like a valid json.

Reply:
I believe `#` is a valid comment in JSON

Reply:
yeah, probably valid for markdown lint :) But it doesn't look to be a standard.